@trycourier/courier-react-components/tsconfig.api-extractor.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "moduleResolution": "node",
```

Comment:
`bundler` resolution was causing api-extractor to find the `*.tsx` files instead of `*.d.ts`. Can fix this and remove the special tsconfig later by generating .d.ts rollups (standard practice) instead of ones for each file.